Orange/tests/test_table.py

```diff
@@ -1277,6 +1277,14 @@ def test_repr_sparse_with_one_row(self):
         table.X = sp.csr_matrix(table.X)
         repr(table)     # make sure repr does not crash
 
+    def test_inf(self):
+        a = np.array([[2,   0,      0,      0],
+                      [0,   np.nan, np.nan, 1],
+                      [0,   0,      np.inf, 0]])
+        with self.assertWarns(Warning):
```

Comment:
Please remove spaces or disable the pylint warning. I now it's stupid, but it pollutes the output on travis.